src/SwaggerExtension/SwaggerExtension/Extensions/SwaggerDependencyInjectionExtensions.cs

```diff
@@ -88,45 +87,21 @@ public static IServiceCollection ConfigureTechBuddySwagger(this IServiceCollecti
     /// <returns>returns app</returns>
     public static IApplicationBuilder UseTechBuddySwagger(this IApplicationBuilder app)
     {
-        return UseTechBuddySwaggerWithApiVersioning(app, null);
-    }
+        IApiVersionDescriptionProvider apiVersioningProvider = app.ApplicationServices.GetService<IApiVersionDescriptionProvider>();
+        SwaggerConfig swaggerConfig = app.ApplicationServices.GetRequiredService<SwaggerConfig>();
 
-    /// <summary>
-    /// This is used to enable Swagger with ApiVersioning Features
-    /// </summary>
-    /// <param name="app">IApplicationBuilder</param>
-    /// <param name="provider">IApiVersionDescriptionProvider</param>
-    /// <returns>returns app</returns>
-    public static IApplicationBuilder UseTechBuddySwaggerWithApiVersioning(this IApplicationBuilder app,
```

Comment:
The name of this method is specified because we want to give the opportunity to implement Swagger with ApiVersioning or not, to the user. If we change the name and implement the IApiVersionDescriptionProvider automatically, we'd take this chance off from the user. So, we have to have this method either with or without the provider parameter. We might have two different methods one of which could request the provider argument whereas other gets it from DI Container.